### PR TITLE
gatherings: separate data files

### LINKED ISSUE #1152: pull speakers  out of gatherings.yml
@jiri-fiala 
it would be much easier to manage the speakers  if they were in a seperate yaml file from the gathering.yml file 

the gatherings file is getting a little long

### PR DESCRIPTION
**Pull request by jiri-fiala**
* separates gathering data into `gatherings.yml`, `speakers.yml` and
  `sponsors.yml` (for sponsors who are not participating in the
  program yet); closes #1152
* fixes Dhwanil Raval's name (removes the dot)

Signed-off-by: Jiri Fiala <jfiala@redhat.com>

### CODE REVIEW

File: data/speakers.yml [hunk 1]
```diff
@@ -0,0 +1,809 @@
...
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/mrunal_patel.jpg	"
+  - id: "urvashi_mohnani"
+    name: "Urvashi Mohnani"
+    role: ""
+    company: "Red Hat"
+    URL: "https://www.redhat.com"
+    intro: ""
+    photo: "speakers/urvashi_mohnani.jpeg	"
```
Comment (wgordon17) [inline data/speakers.yml]: @jiri-fiala Since this is your PR...would you mind being the one to clean up these extra spaces?

**Comment by jiri-fiala** [comment]
I agree re the workflow. The current one introduced some errors and pollutes the commit history; I'd prefer the standard PR reviewed by someone else and squashed before merging. We may want postponing this one until @dmueller2001 confirms it.

**Comment by dmueller2001** [comment]
just saw this today..apologies for the delay. will i lose any of the speakers that i have added since you made this pr if we merge it as i have added/edited quite a few..if so, let's wait until post Kubecon/EU (May 21) before we break things..

**Comment by jiri-fiala** [comment]
@dmueller2001 makes sense; I'll rebase this around May 22nd and will get back to you

**Pull request merged**